packages/components/app/styles/components/tabs.scss

```diff
@@ -30,6 +30,7 @@ $indicator-height: 3px;
   position: relative; // for positioning temp indicator
 
   // temporary indicator
+  /* TODO: refactor
```

Comment:
I commented this part out for now until I'm ready to start implementing the actual selector.